analyzer/src/main/kotlin/managers/NuGet.kt

```diff
@@ -39,6 +39,22 @@ import com.here.ort.model.config.RepositoryConfiguration
 
 import java.io.File
 
+// See https://docs.microsoft.com/en-us/nuget/reference/packages-config.
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I wonder if the names should be made less generic now. Did you consider to keep them inside the NuGet class but move them out of the static context?

Reply:
Good point, for some reason I missed to consider that option. Will do.